### DISCUSSION TITLE
Capitalization on replies

### DISCUSSION THREAD

**Discussion by DavidToddMcCarty** [mastodon/mastodon#20590] [General]
I was wondering if there was any way to default to a reply, to begin with a capital letter just like it does with a new post. Seems like an easy fix, no?

**Comment by trwnh** [comment]
this would depend on your keyboard's auto-capitalization algorithm -- i don't think there's anything mastodon can do regarding this

**Comment by DavidToddMcCarty** [comment]
﻿It works that way on Twitter and Facebook. Not sure what my configuration would have to do with it. But I don’t know either. 
--
David Todd McCarty
THE STANDARD
http://www.capemaystandard.com
609.602.5228 [m]

**Reply by trwnh** [reply]
Twitter and Facebook have an empty text field, but Mastodon starts with a mention, which I guess is being detected as the start of a sentence by whichever keyboard you are using (Apple, Google, etc).

**Comment by DavidToddMcCarty** [comment]
Yes. That seems like poor design. Why does the message body need to start with the addressee? Why can’t it be more hidden? I not an engineer. Just thinking from a UI perspective. 

--
David Todd McCarty
THE STANDARD
http://www.capemaystandard.com
609.602.5228 [m]

**Comment by ineffyble** [comment]
I've created an issue/suggestion here: https://github.com/mastodon/mastodon/issues/20618
